### DISCUSSION TITLE
Misleading Header in Audio Gear

### DISCUSSION THREAD

**Discussion by ecb344** [ITPNYU/booking-app#246] [General]
Hi! May we please remove this header:
https://sites.google.com/nyu.edu/370jmediacommons/rental-inventory/audio?authuser=0#h.vjbm3265305r

The items below it are apart of the mobile audio rack kits, and I think this header makes it appear as thought the items below are part of a separate "Sennheiser EW-DX Wireless System" when in fact these items are not available to be rented separately from the mobile audio rack kit.

**Comment by nopivnick** [comment]
cc: @ss12430 

thanks for your feedback. these are issues with the Media Commons website design, not the booking tool, and so out of scope for this project.
